app/javascript/elm/tests/MainTests.elm

```diff
@@ -887,11 +887,11 @@ updateTests =
                     |> update (UpdateHeatMapMaximum int)
                     |> Tuple.first
                     |> Expect.equal expected
-        , fuzz bool "ToggleIsSearchOn changes the state of model.isSearchOn" <|
+        , fuzz bool "ToggleIsSearchOn changes the state of model.isSearchAsIMoveOn" <|
             \onOrOff ->
-                { defaultModel | isSearchOn = onOrOff }
+                { defaultModel | isSearchAsIMoveOn = onOrOff }
```

Comment:
not sure why but `onOrOff` warms my heart 😍 but above you have `isSearchAsIMoveOn` in a similar test, I'd do this or that in both places